wiki/Ranking_Criteria/osu!catch/en.md

```diff
@@ -1,21 +1,22 @@
+osu!catch Ranking Criteria
+=============================
+
 The **Catch the Beat Ranking Criteria** is a set of rules and guidelines that apply to the creation of Catch the Beat-specific difficulties. In order to get a Catch the Beat-specific difficulty ranked, it is mandatory the creation obeys to the listed criteria. While **all rules must be followed in any circumstance**, guidelines may be violated under exceptional circumstances. These exceptional circumstances must be warranted by an exhaustive explanation as of why the guideline has been violated and why not violating it will interfere with the overall quality of the creation.
 
 Glossary
-========
+---------
 
 These terms are frequently appearing when getting in touch with Catch the Beat difficulties and are also used within the Ranking Criteria.
 
-Difficulty Names
-----------------
+### Difficulty Names
 
--   ![](Easy-f.png "fig:Easy-f.png") **Cup**. Easy
--   ![](Normal-f.png "fig:Normal-f.png") **Salad**. Normal
--   ![](Hard-f.png "fig:Hard-f.png") **Platter**. Hard
--   ![](Insane-f.png "fig:Insane-f.png") **Rain**. Insane
--   ![](Expert-f.png "fig:Expert-f.png") **Overdose**. Expert
+-   ![Easy icon](/wiki/shared/Easy-f.png "Easy icon") **Cup**. Easy
+-   ![Normal icon](/wiki/shared/Normal-f.png "Normal icon") **Salad**. Normal
+-   ![Hard icon](/wiki/shared/Hard-f.png "Hard icon") **Platter**. Hard
+-   ![Insane icon](/wiki/shared/Insane-f.png "Insane icon") **Rain**. Insane
+-   ![Expert icon](/wiki/shared/Expert-f.png "Expert icon") **Overdose**. Expert
 
-Gameplay
---------
+### Gameplay
 
 -   **Fruit.** A large object represented by a hitcircle, slider head, tail or repeat.
```

Comment:
For this and other terms below:
- `**Fruit:**`
- `**Drop:**`
- `**Droplet:**`
- `**Banana:**`
- `**Jump:**`
- `**Hyperjump:**`
- `**Edge Dash:**`
- `**Trigger Distance:**`